config/brakeman.ignore

```diff
@@ -0,0 +1,26 @@
+{
+  "ignored_warnings": [
+    {
+      "warning_type": "Dynamic Render Path",
+      "warning_code": 15,
+      "fingerprint": "aa162721dfe63daa1c04c888b19e85ba75855583622c20572118575e01c9426b",
+      "check_name": "Render",
+      "message": "Render path contains parameter value",
+      "file": "app/controllers/devices_guidance_controller.rb",
```

Comment:
I'm a little uneasy about disabling brakeman warnings, but I think in this case, given that it only allows the subpage IDs that are present as keys in the particular section of the i18n file, there is sufficient protection against code injection.

Reply:
Might be an idea to add a feature spec to `spec/features/devices_guidance_spec.rb` that tests that scenario (requesting a subpage that does not exist responds with a 404)?

Reply:
Agreed – added in ad4f9b9